### PR TITLE
Attempt to enable releasing the BOM by creating a dummy github tag

### PR DESCRIPTION
**Pull request by geoand**
Relates to: SB-810

### CODE REVIEW

**Comment by geoand** [comment]
I can't test the actual release obviously :(

**Comment by gytis** [comment]
@geoand I thought Atomist was supposed to do this type of tasks?

**Comment by geoand** [comment]
That's a good point, but until we gain some good experience at running it,
we are just using it locally (which means that everything needs to be set
up on the developers machine), which is why for the time being I want to
setup CircleCI to be able to create releases.

Once that is done,  Atomist can just create the tag and CircleCI can take
over from there.

**Comment by geoand** [comment]
@gytis To continue on the previous thought and your question, when we start running Atomist on a server (with it's own environment setup for our needs), then yes, it can take care of performing the Maven release.

For the moment I just want us to have solution that doesn't rely on having our machines correctly setup. It's not the perfect solution, but lots of projects use it and I think it's a good stepping stone

**Comment by gytis** [comment]
@geoand got you

**Comment by geoand** [comment]
I'm flagging this as WIP because there are a few things I can try without actually making a release

File: .circleci/config.yml [hunk 1]
```diff
@@ -124,14 +124,37 @@ jobs:
...
+    docker:
+    - image: circleci/openjdk:8-jdk-browsers
+
+    steps:
+    - checkout
+
+    - run:
+        name: Perform release
+        command: |
+          tagRE='release-(.*)'
```
Comment (metacosm) [inline .circleci/config.yml]: We might want to make this more precise in future versions.

Comment (geoand) [reply .circleci/config.yml]: Indeed

**Pull request merged**